x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -238,25 +228,60 @@ export const createIncident = async ({
   let comments: ExternalServiceComment[] = [];
 
   if (commentsToBeUpdated && Array.isArray(commentsToBeUpdated) && commentsToBeUpdated.length > 0) {
-    const commentsMapping = mappings.find((m) => m.source === 'comments');
-    if (commentsMapping?.action_type !== 'nothing') {
-      comments = transformComments(commentsToBeUpdated, ['informationAdded'], userProfiles);
-    }
+    comments = addKibanaInformationToComments(commentsToBeUpdated, userProfiles);
   }
 
-  comments = addAlertMessage(theCase.id, caseComments, comments);
+  comments = addAlertMessage(theCase.id, theCase.comments, comments);
+  return comments;
+};
 
-  return { incident, comments };
+export const addKibanaInformationToDescription = (
+  theCase: CaseResponse,
+  userProfiles?: Map<string, UserProfile>,
+  publicBaseUrl?: IBasePath['publicBaseUrl']
+) => {
+  const descriptionWithKibanaInformation = `${theCase.description}\n\nAdded by ${getEntity(
+    {
+      createdBy: theCase.created_by,
+      updatedBy: theCase.updated_by,
+    },
+    userProfiles
+  )}.`;
+
+  if (!publicBaseUrl) {
+    return descriptionWithKibanaInformation;
+  }
+
+  const caseUrl = getCaseViewPath({ publicBaseUrl, caseId: theCase.id, owner: theCase.owner });
+
+  return `${descriptionWithKibanaInformation}\nFor more details view this case in Kibana.\nCase URL: ${caseUrl}`;
```

Comment:
This would read better with a `,` after `details` 

```
For more details, view this case in Kibana.
```

or consider making the link part of the message

```
For more details, <<view this case in Kibana>>.
```

And can that message be message be internationalized?

Reply:
Also testing on IBM Security QRadar SOAR, the text isn't rendered as a link (whereas it is in e.g. Jira)

Reply:
Each external service renders the content differently. All external services escape the content before showing it for security reasons. Some of them support the markdown format and some of them have their own syntax (Jira for example). For this reason, we decided with @shanisagiv1 to send only the URL as text and leave the interpretation of the link to the external services. Btw, I added the comma to the text and I translated the message.

Reply:
@peteharverson @lcawl I added the following text in the total alerts comments. What do you think?

<img width="831" alt="Screenshot 2022-10-13 at 12 02 57 PM" src="https://user-images.githubusercontent.com/7871006/195553267-4786181e-8787-4d91-a2a2-5ecaad85e435.png">

Reply:
Thanks for the clarification @cnasikas  - I see we are limited on formatting the link here as it will be the external service that displays the content. So the change to the text, and internationalization, LGTM.